src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -1237,16 +1247,81 @@ impl A320HydraulicBrakingLogic {
 
 impl SimulationElement for A320HydraulicBrakingLogic {
     fn read(&mut self, state: &mut SimulatorReader) {
-        self.parking_brake_demand = state.read_bool("BRAKE PARKING INDICATOR");
+        self.parking_brake_demand = state.read_bool("BRAKE PARKING POSITION");
```

Comment:
**suggestion**: For future reference I suggest adding information as to how `L:A32NX_PARK_BRAKE_LEVER_POS`, `BRAKE PARKING POSITION` and `BRAKE PARKING INDICATOR` relate. This will probably be very helpful for others trying to understand how different elements connect.

Reply:
This is indeed not clear for sure and counter intuitive as we basically receive outputs event and write the inputs....   Will try to explain clearly :p
Adding some comments but should I create a side document to centralize the system event handling ?

Reply:
Might it be that this code shouldn't be here?

It seems you always read:
```rust
self.park_brake_lever_is_set = state.read_bool("BRAKE PARKING POSITION");
```
and then write the value you read:
```rust
writer.write_bool("PARK_BRAKE_LEVER_POS", self.park_brake_lever_is_set);
```

You could consider moving that to the MSFS specific layer, as it sounds to me like "tricks" you're doing to make MSFS work. It doesn't sound like something that could apply to _any_ simulation.

Reply:
Definitly a msfs trick from masking those inputs.... Will think about this thanks!

Reply:
Went by short circuiting the variables write directly in the Brake structure masking those inputs. Unsure if it is relevant to use the reader/writer mechanism for that masked inputs case?

Reply:
Looks good the way you did it! I think we shouldn't overly limit what the handlers are allowed to do.